src/nccl_ofi_rdma.c

```diff
@@ -518,6 +519,10 @@ static inline int get_properties(nccl_net_ofi_device_t *base_dev,
 	props->rma_supported = 1;
 	assert(is_max_write_inline_size_initialized);
 	props->max_write_inline_size = max_write_inline_size;
+	props->dmabuf_support = ((info->caps & FI_HMEM) != 0) &&
```

Comment:
why is this in the per-protocol bits instead of the shared init in net.c?

Reply:
there's no shared notion of the selected API version